### PR TITLE
🙈 slightly improve .gitignore file

### PR DESCRIPTION
**Pull request by gmarabout**
# Goal
Ignore common or intermediary files (mostly the `node_modules` in the `gui` folder).

### CODE REVIEW

**Review by toan-quach — APPROVED** [review]
lgtm!

File: .gitignore [hunk 1]
```diff
@@ -1,11 +1,17 @@
-.idea
```
Comment (jrobinAV) [inline .gitignore:1]: Why remove the .idea?

File: .gitignore [hunk 2]
```diff
@@ -22,3 +28,6 @@ dist/
 # demo files
 demo-*
 demo_*/
+
+# Gui node_modules
+gui/node_modules
```
Comment (gmarabout) [inline .gitignore:33]: **note** : not useful after all. The `gui` folder is a reminiscence from the past and should no longer be there.

Comment (FredLL-Avaiga) [reply .gitignore:33]: Will you remove the line then ?

**Review by FredLL-Avaiga — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (gmarabout) [reply .gitignore:1]: Because it's duplicated.

File: .gitignore (hunk 2 above)
Comment (gmarabout) [reply .gitignore:33]: Yes. Not sure this PR makes sense any longer. closing it.

**Pull request closed without merging**